Sources/Internal/FrameProvider.swift

```diff
@@ -77,6 +79,16 @@ final class FrameProvider {
       let x = minXOfMonth(containingItemWithFrame: layoutItem.frame, at: position)
       let y = minYOfMonth(containingDayItemWithFrame: layoutItem.frame, atRowInMonth: rowInMonth)
       return CGPoint(x: x, y: y)
+        
+    case .monthFooter(let month):
+      let x = layoutItem.frame.origin.x
+      let y = layoutItem.frame.minY -
+        monthHeaderHeight -
+        content.monthDayInsets.top -
+        (monthsLayout.pinDaysOfWeekToTop ? 0 : (daySize.height + content.verticalDayMargin)) -
+        CGFloat((CGFloat(numberOfRows(in: month)) * daySize.height) + (CGFloat(numberOfRows(in: month) - 1) * content.verticalDayMargin)) -
+        content.monthDayInsets.bottom
+      return CGPoint(x: x, y: y)
```

Comment:
```suggestion
    case .monthFooter(let month):
      let numberOfWeekRows = self.numberOfWeekRows(in: month)

      let x = layoutItem.frame.origin.x
      let y = layoutItem.frame.minY -
        content.monthDayInsets.bottom -
        heightOfDayContent(forNumberOfWeekRows: numberOfWeekRows) -
        heightOfDaysOfTheWeekRowInMonth() -
        content.monthDayInsets.top -
        monthHeaderHeight
      return CGPoint(x: x, y: y)
```

Adopting new helper functions and swapping the order of subtraction so it reads like we'd subtracting each adjacent piece upwards until we get to our month's min Y value.